crates/ruff/src/registry.rs

```diff
@@ -7,7 +7,7 @@ pub use codes::Rule;
 use ruff_macros::RuleNamespace;
 pub use rule_set::{RuleSet, RuleSetIterator};
 
-use crate::codes::{self, RuleCodePrefix};
+use crate::codes::{self};
```

Comment:
nit:
```suggestion
use crate::codes;
```
(i'm surprised clippy doesn't complain about this)